graviti/workspace.py

```diff
@@ -18,8 +19,8 @@ class Workspace:
 
     """
 
-    _DEFAULT_URL_CN = "https://api.graviti.cn/"
-    _DEFAULT_URL_COM = "https://api.graviti.com/"
+    _DEFAULT_URL_CN = "https://api.graviti.cn"
+    _DEFAULT_URL_COM = "https://api.graviti.com"
```

Comment:
but the url in openapi example need to add `/`. Do we need to modify them?

Reply:
Does it really need the `/`
```python
In [2]: urljoin("https://api.graviti.com", "xxx")
Out[2]: 'https://api.graviti.com/xxx'
```